PaintTests/WinTests.cpp

```diff
@@ -609,7 +609,7 @@ namespace PaintTests
 			Renderer* rendererPtr = &dummyRenderer;
 			uint8_t ColA = 0;
```

Comment:
Should be camelCase, not PascalCase
**SHOULD**

Reply:
done